packages/geoprocessing/scripts/init/createReport.ts

```diff
@@ -0,0 +1,317 @@
+import inquirer from "inquirer";
+import ora from "ora";
+import fs from "fs-extra";
+import path from "path";
+import chalk from "chalk";
+import camelcase from "camelcase";
+import {
+  ExecutionMode,
+  metricGroupsSchema,
+  GeoprocessingJsonConfig,
+} from "../../src/types";
+import {
+  getBlankComponentPath,
+  getBlankFunctionPath,
+  getOceanEEZComponentPath,
+  getOceanEEZFunctionPath,
+  getProjectComponentPath,
+  getProjectConfigPath,
+  getProjectFunctionPath,
+} from "../util/getPaths";
+
+// CLI questions
+const createReport = async () => {
+  // Report type, description, and execution mode
+  const answers = await inquirer.prompt([
+    {
+      type: "list",
+      name: "type",
+      message: "Type of report to create",
+      choices: [
+        {
+          value: "blank",
+          name: "Blank report",
+        },
+        {
+          value: "raster",
+          name: "Raster overlap report - Calculates sketch overlap with raster data sources",
+        },
+        {
+          value: "vector",
+          name: "Vector overlap report - Calculates sketch overlap with vector data sources",
+        },
+      ],
+    },
+    {
+      type: "input",
+      name: "description",
+      message: "Describe what this report calculates",
+    },
+    {
+      type: "list",
+      name: "executionMode",
+      message: "Choose an execution mode for this report",
+      choices: [
+        {
+          value: "sync",
+          name: "Sync - Best for quick analyses (< 2s)",
+        },
+        {
+          value: "async",
+          name: "Async - Better for long-running processes",
+        },
+      ],
+    },
+  ]);
+
+  // Title of report
+  if (answers.type === "raster" || answers.type === "vector") {
+    // For raster and vector overlap reports, we need to know which metric group to report on
+    const rawMetrics = fs.readJSONSync(
+      `${getProjectConfigPath("")}/metrics.json`
+    );
+    const metrics = metricGroupsSchema.parse(rawMetrics);
+    const geoprocessingJson = JSON.parse(
+      fs.readFileSync("./geoprocessing.json").toString()
+    ) as GeoprocessingJsonConfig;
+    const gpFunctions = geoprocessingJson.geoprocessingFunctions || [];
+    const availableMetricGroups = metrics
+      .map((metric) => metric.metricId)
+      .filter(
+        (metricId) => !gpFunctions.includes(`src/functions/${metricId}.ts`)
+      );
+    if (!availableMetricGroups.length) {
+      console.log(
+        "No available metric groups - create a metric group to continue"
```

Comment:
Is overwriting bad?  Consider just warning them that an existing report will be overwritten.

If leaving it as-is, is the message complete.  Is it also possible that they need to delete an existing report like "No available metric groups - create a new metric group to continue or delete an existing report, then run this command again to recreate it."

Reply:
The idea behind not overwriting was to avoid overwriting a report that's been edited/customized, and that the most likely use of `create:report` as being right after creating a new metric group. I can add to the message

Reply:
Cool, the message update looks good.  I don't have anything else.